gradle.properties

```diff
@@ -1,6 +1,6 @@
 kotlin.code.style=official
 
-org.gradle.jvmargs=-Duser.country=US -Dkotlin.daemon.jvm.options=-Xmx3072m -Dfile.encoding=UTF-8 -Xmx4096m
+org.gradle.jvmargs=-Duser.country=US -Dkotlin.daemon.jvm.options=-Xmx4096m -Dfile.encoding=UTF-8 -Xmx4096m
```

Comment:
Apparently we were right on the edge of maxing out the compiler's memory before; without this I get out-of-memory errors from the build.